meson_options.txt

```diff
@@ -18,6 +18,7 @@ option('plugin_uefi', type : 'boolean', value : true, description : 'enable UEFI
 option('plugin_nvme', type : 'boolean', value : true, description : 'enable NVMe support')
 option('systemd', type : 'boolean', value : true, description : 'enable systemd support')
 option('systemdunitdir', type: 'string', value: '', description: 'Directory for systemd units')
+option('elogind', type : 'boolean', value : true, description : 'enable elogind support')
 option('tests', type : 'boolean', value : true, description : 'enable tests')
```

Comment:
So this is making elogind required by default by setting this true. I would recommend to make this default to false since elogind is less common than systemd.

Reply:
I agree, this should be false by default. Fixed.